src/isIsogram.test.js

```diff
@@ -6,4 +6,41 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  test(`word is **isogram** & consisted of Lower&UpperCases`, () => {
```

Comment:
Please, provide more informative description what you check in this test (start with 'should...')

Reply:
And fix it in all test descriptions